fv3gfs-physics/fv3gfs/physics/stencils/microphysics.py

```diff
@@ -653,7 +667,7 @@ def warm_rain(
 
         # Auto-conversion assuming linear subgrid vertical distribution of
         # cloud water following lin et al. 1994, mwr
-        if constants.irain_f != 0:
+        if irain_f != 0:
```

Comment:
Change to boolean

Reply:
I think people have namelists with this as 0 or 1